src/Microsoft.DotNet.Interactive.CSharp/AspNetKernelExtensions.cs

```diff
@@ -0,0 +1,169 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.CSharp
+{
+    public static class AspNetKernelExtensions
+    {
+        public static CSharpKernel UseAspNet(
+            this CSharpKernel kernel)
+        {
+            var command = new SubmitCode(@"
+Environment.SetEnvironmentVariable($""ASPNETCORE_{WebHostDefaults.PreventHostingStartupKey}"", ""true"");
+
+class LogLevelMonitor : IOptionsMonitor<LoggerFilterOptions>
```

Comment:
We can likely implement these things in library code, possibly as an `IKernelExtension`, rather than using a code submission.